### DISCUSSION TITLE
New state modifier

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#4428] [Ideas]
When styling input fields, I want to prompt the user that their input are invalid by changing the style of the field.
Maybe something like `invalid:bg-red-100` just like hover and focus state modifiers

**Comment by jonathanstanley** [comment]
added in https://github.com/tailwindlabs/tailwindcss/pull/4482

**Reply by reinink** [reply]
Yup, exactly. You can find an example of this in the docs [here](https://tailwindcss.com/docs/hover-focus-and-other-states#form-states) and [here](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state).
